doc/changes/changes_3.0.0.md

```diff
@@ -0,0 +1,20 @@
+# Test Database Builder 3.0.0, released 2021-01-XX
+
+Code name: Bulk insert for tables
+
+## Summary
+
+In this release, we removed the `Table#getRows()` method and by that cause an API change. We removed the method, since it requires all rows to be kept in memory which limits the size of the tables that can be built.
```

Comment:
```suggestion
In this release, we removed the `Table#getRows()` method and by that cause an API change. We removed this method, since it requires all rows to be kept in memory which limits the size of the tables that can be built.
```